### ISSUE TITLE
move_group Process Dying in MoveIt2 Tutorials

### ISSUE THREAD

**Issue by Leopold-Klotz** [moveit/moveit2#2887]
Hello all,

- Ubuntu: 22.04.4 LTS
- ROS2: Humble

I was having an issue with move_group crashing when I was trying to launch a personal package which depended on moveit2. I tried to figure it out, but eventually could not find the root of the issue. Initially I thought that it must be something with my machine, but with a fresh install I'm now having issues with the provided moveit2 tutorials as well.

To try and find the issue, I have just freshly installed ubuntu 22.04 on my machine, and a new installation of both ROS2 Humble following the main instructions, as well as installing MoveIt2. When I try and go through the tutorial, I am getting an error where the move_group is process dying. 

Any advice would be appreciated and if I figure it out I will post the solution/fix the code.
From what I can tell, the relevant logs are:

[move_group-4] terminate called after throwing an instance of 'rclcpp::ParameterTypeException'
[move_group-4]   what():  expected [string_array] got [string]
[move_group-4] Stack trace (most recent call last):
[move_group-4] #17   Object "", at 0xffffffffffffffff, in 
[move_group-4] #16   Object "/home/leo/ws_moveit/install/moveit_ros_move_group/lib/moveit_ros_move_group/move_group", at 0x5b58c1313094, in _start
[move_group-4] #15   Source "../csu/libc-start.c", line 392, in __libc_start_main_impl [0x7efb48629e3f]
[move_group-4] #14   Source "../sysdeps/nptl/libc_start_call_main.h", line 58, in __libc_start_call_main [0x7efb48629d8f]
[move_group-4] #13   Object "/home/leo/ws_moveit/install/moveit_ros_move_group/lib/moveit_ros_move_group/move_group", at 0x5b58c131221e, in main
[move_group-4] #12   Object "/home/leo/ws_moveit/install/moveit_ros_planning/lib/libmoveit_cpp.so.2.10.0", at 0x7efb492b2287, in moveit_cpp::MoveItCpp::MoveItCpp(std::shared_ptr<rclcpp::Node> const&, moveit_cpp::MoveItCpp::Options const&)
[move_group-4] #11   Object "/home/leo/ws_moveit/install/moveit_ros_planning/lib/libmoveit_cpp.so.2.10.0", at 0x7efb492b128b, in moveit_cpp::MoveItCpp::loadPlanningPipelines(moveit_cpp::MoveItCpp::PlanningPipelineOptions const&)
[move_group-4] #10   Object "/home/leo/ws_moveit/install/moveit_ros_planning/lib/libmoveit_planning_pipeline_interfaces.so.2.10.0", at 0x7efb488fac76, in moveit::planning_pipeline_interfaces::createPlanningPipelineMap(std::vector<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::allocator<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > > > const&, std::shared_ptr<moveit::core::RobotModel const> const&, std::shared_ptr<rclcpp::Node> const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)
[move_group-4] #9    Object "/home/leo/ws_moveit/install/moveit_ros_planning/lib/libmoveit_planning_pipeline.so.2.10.0", at 0x7efb4886cd26, in planning_pipeline::PlanningPipeline::PlanningPipeline(std::shared_ptr<moveit::core::RobotModel const> const&, std::shared_ptr<rclcpp::Node> const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)
[move_group-4] #8    Object "/home/leo/ws_moveit/install/moveit_ros_planning/lib/libmoveit_planning_pipeline.so.2.10.0", at 0x7efb488b10b3, in planning_pipeline_parameters::ParamListener::declare_params()
[move_group-4] #7    Object "/opt/ros/humble/lib/librclcpp.so", at 0x7efb48eb2849, in 
[move_group-4] #6    Object "/usr/lib/x86_64-linux-gnu/libstdc++.so.6.0.30", at 0x7efb48aae4d7, in __cxa_throw
[move_group-4] #5    Object "/usr/lib/x86_64-linux-gnu/libstdc++.so.6.0.30", at 0x7efb48aae276, in std::terminate()
[move_group-4] #4    Object "/usr/lib/x86_64-linux-gnu/libstdc++.so.6.0.30", at 0x7efb48aae20b, in 
[move_group-4] #3    Object "/usr/lib/x86_64-linux-gnu/libstdc++.so.6.0.30", at 0x7efb48aa2b9d, in 
[move_group-4] #2    Source "./stdlib/abort.c", line 79, in abort [0x7efb486287f2]
[move_group-4] #1    Source "../sysdeps/posix/raise.c", line 26, in raise [0x7efb48642475]
[move_group-4] #0  | Source "./nptl/pthread_kill.c", line 89, in __pthread_kill_internal
[move_group-4]     | Source "./nptl/pthread_kill.c", line 78, in __pthread_kill_implementation
[move_group-4]       Source "./nptl/pthread_kill.c", line 44, in __pthread_kill [0x7efb486969fc]
[move_group-4] Aborted (Signal sent by tkill() 174093 1000)
[ERROR] [move_group-4]: process has died [pid 174093, exit code -6, cmd '/home/leo/ws_moveit/install/moveit_ros_move_group/lib/moveit_ros_move_group/move_group --ros-args --params-file /tmp/launch_params_yoqu6zsx'].

**Comment by JenniferBuehler** [comment]
I confirm that I am experiencing the same. It's the same issue as [here](https://github.com/moveit/moveit2/issues/2854#issuecomment-2152224079) and this solution solved it for me.

**Comment by manadaniel** [comment]
I am experiencing the same issue with the tutorials and other projects using the following setup:

- Ubuntu 24.04
- ROS2 Jazzy
- Moveit2 Main on commit 94796046b5ae6fc2a8e5f5ee73db4944d71a5643

Here is the terminal output:

```
[INFO] [launch]: Default logging verbosity is set to INFO
[INFO] [rviz2-1]: process started with pid [5760]
[INFO] [static_transform_publisher-2]: process started with pid [5761]
[INFO] [robot_state_publisher-3]: process started with pid [5762]
[INFO] [move_group-4]: process started with pid [5763]
[INFO] [ros2_control_node-5]: process started with pid [5764]
[INFO] [spawner-6]: process started with pid [5765]
[INFO] [spawner-7]: process started with pid [5766]
[INFO] [spawner-8]: process started with pid [5767]
[static_transform_publisher-2] [INFO] [1719469827.767164578] [static_transform_publisher]: Spinning until stopped - publishing transform
[static_transform_publisher-2] translation: ('0.000000', '0.000000', '0.000000')
[static_transform_publisher-2] rotation: ('0.000000', '0.000000', '0.000000', '1.000000')
[static_transform_publisher-2] from 'world' to 'base_link'
[robot_state_publisher-3] [INFO] [1719469827.774529923] [robot_state_publisher]: Robot initialized
[ros2_control_node-5] [INFO] [1719469827.784998003] [controller_manager]: Subscribing to '/robot_description' topic for robot description.
[ros2_control_node-5] [INFO] [1719469827.785051080] [controller_manager]: update rate is 1000 Hz
[ros2_control_node-5] [WARN] [1719469827.785092366] [controller_manager]: Could not enable FIFO RT scheduling policy. Consider setting up your user to do FIFO RT scheduling. See [https://control.ros.org/master/doc/ros2_control/controller_manager/doc/userdoc.html] for details.
[ros2_control_node-5] [INFO] [1719469827.794998360] [controller_manager]: Received robot description from topic.
[ros2_control_node-5] Error parsing the limits for the interface: position from the tags [min: '{-2*pi}' and max: '{2*pi}'] within ros2_control tag inside the URDF. Skipping it
[ros2_control_node-5] Error parsing the limits for the interface: position from the tags [min: '{-2*pi}' and max: '{2*pi}'] within ros2_control tag inside the URDF. Skipping it
[ros2_control_node-5] Error parsing the limits for the interface: position from the tags [min: '{-2*pi}' and max: '{2*pi}'] within ros2_control tag inside the URDF. Skipping it
[ros2_control_node-5] Error parsing the limits for the interface: position from the tags [min: '{-2*pi}' and max: '{2*pi}'] within ros2_control tag inside the URDF. Skipping it
[ros2_control_node-5] Warning: Mimic joints defined in ros2_control tag are deprecated. Please define mimic joints in URDF.
[ros2_control_node-5] Warning: Mimic joints defined in ros2_control tag are deprecated. Please define mimic joints in URDF.
[ros2_control_node-5] Warning: Mimic joints defined in ros2_control tag are deprecated. Please define mimic joints in URDF.
[ros2_control_node-5] Warning: Mimic joints defined in ros2_control tag are deprecated. Please define mimic joints in URDF.
[ros2_control_node-5] Warning: Mimic joints defined in ros2_control tag are deprecated. Please define mimic joints in URDF.
[move_group-4] [INFO] [1719469827.797194525] [move_group.moveit.moveit.ros.rdf_loader]: Loaded robot model in 0.00319745 seconds
[ros2_control_node-5] [INFO] [1719469827.797288718] [resource_manager]: Loading hardware 'KortexMultiInterfaceHardware' 
[move_group-4] [INFO] [1719469827.797424814] [move_group.moveit.moveit.core.robot_model]: Loading robot model 'gen3'...
[move_group-4] [INFO] [1719469827.797432949] [move_group.moveit.moveit.core.robot_model]: No root/virtual joint specified in SRDF. Assuming fixed joint
[ros2_control_node-5] [INFO] [1719469827.798029218] [resource_manager]: Loaded hardware 'KortexMultiInterfaceHardware' from plugin 'mock_components/GenericSystem'
[ros2_control_node-5] [INFO] [1719469827.798041069] [resource_manager]: Initialize hardware 'KortexMultiInterfaceHardware' 
[ros2_control_node-5] [INFO] [1719469827.798057308] [resource_manager]: Successful initialization of hardware 'KortexMultiInterfaceHardware'
[ros2_control_node-5] [INFO] [1719469827.798068156] [resource_manager]: Loading hardware 'RobotiqGripperHardwareInterface' 
[ros2_control_node-5] [INFO] [1719469827.798070907] [resource_manager]: Loaded hardware 'RobotiqGripperHardwareInterface' from plugin 'mock_components/GenericSystem'
[ros2_control_node-5] [INFO] [1719469827.798072793] [resource_manager]: Initialize hardware 'RobotiqGripperHardwareInterface' 
[ros2_control_node-5] [INFO] [1719469827.798080331] [resource_manager]: Successful initialization of hardware 'RobotiqGripperHardwareInterface'
[ros2_control_node-5] [INFO] [1719469827.798105786] [resource_manager]: 'configure' hardware 'RobotiqGripperHardwareInterface' 
[ros2_control_node-5] [INFO] [1719469827.798108834] [resource_manager]: Successful 'configure' of hardware 'RobotiqGripperHardwareInterface'
[ros2_control_node-5] [INFO] [1719469827.798111362] [resource_manager]: 'activate' hardware 'RobotiqGripperHardwareInterface' 
[ros2_control_node-5] [INFO] [1719469827.798112699] [resource_manager]: Successful 'activate' of hardware 'RobotiqGripperHardwareInterface'
[ros2_control_node-5] [INFO] [1719469827.798114127] [resource_manager]: 'configure' hardware 'KortexMultiInterfaceHardware' 
[ros2_control_node-5] [INFO] [1719469827.798115191] [resource_manager]: Successful 'configure' of hardware 'KortexMultiInterfaceHardware'
[ros2_control_node-5] [INFO] [1719469827.798118062] [resource_manager]: 'activate' hardware 'KortexMultiInterfaceHardware' 
[ros2_control_node-5] [INFO] [1719469827.798119210] [resource_manager]: Successful 'activate' of hardware 'KortexMultiInterfaceHardware'
[ros2_control_node-5] [INFO] [1719469827.801235322] [controller_manager]: Received robot description from topic.
[ros2_control_node-5] [WARN] [1719469827.801259812] [controller_manager]: ResourceManager has already loaded an urdf file. Ignoring attempt to reload a robot description file.
[move_group-4] [INFO] [1719469827.865242124] [move_group.moveit.moveit.kinematics.kdl_kinematics_plugin]: Joint weights for group 'manipulator': 1 1 1 1 1 1 1
[ros2_control_node-5] [INFO] [1719469827.960411290] [controller_manager]: Loading controller 'robotiq_gripper_controller'
[spawner-8] [INFO] [1719469827.972736530] [spawner_robotiq_gripper_controller]: Loaded robotiq_gripper_controller
[ros2_control_node-5] [INFO] [1719469827.973183300] [controller_manager]: Configuring controller 'robotiq_gripper_controller'
[ros2_control_node-5] [INFO] [1719469827.973231954] [robotiq_gripper_controller]: Action status changes will be monitored at 20Hz.
[spawner-8] [INFO] [1719469827.983265935] [spawner_robotiq_gripper_controller]: Configured and activated robotiq_gripper_controller
[move_group-4] [INFO] [1719469828.028805657] [move_group.moveit.moveit.ros.planning_scene_monitor]: Publishing maintained planning scene on 'monitored_planning_scene'
[move_group-4] [INFO] [1719469828.028887197] [move_group.moveit.moveit.ros.moveit_cpp]: Listening to 'joint_states' for joint states
[move_group-4] [INFO] [1719469828.029226693] [move_group.moveit.moveit.ros.current_state_monitor]: Listening to joint states on topic 'joint_states'
[move_group-4] [INFO] [1719469828.029366729] [move_group.moveit.moveit.ros.planning_scene_monitor]: Listening to '/attached_collision_object' for attached collision objects
[move_group-4] [INFO] [1719469828.029371889] [move_group.moveit.moveit.ros.planning_scene_monitor]: Stopping existing planning scene publisher.
[move_group-4] [INFO] [1719469828.029436666] [move_group.moveit.moveit.ros.planning_scene_monitor]: Stopped publishing maintained planning scene.
[move_group-4] [INFO] [1719469828.029538662] [move_group.moveit.moveit.ros.planning_scene_monitor]: Publishing maintained planning scene on 'monitored_planning_scene'
[move_group-4] [INFO] [1719469828.029568132] [move_group.moveit.moveit.ros.planning_scene_monitor]: Starting planning scene monitor
[move_group-4] [INFO] [1719469828.029668058] [move_group.moveit.moveit.ros.planning_scene_monitor]: Listening to '/planning_scene'
[move_group-4] [INFO] [1719469828.029672202] [move_group.moveit.moveit.ros.planning_scene_monitor]: Starting world geometry update monitor for collision objects, attached objects, octomap updates.
[move_group-4] [INFO] [1719469828.029899249] [move_group.moveit.moveit.ros.planning_scene_monitor]: Listening to 'collision_object'
[move_group-4] [INFO] [1719469828.029990137] [move_group.moveit.moveit.ros.planning_scene_monitor]: Listening to 'planning_scene_world' for planning scene world geometry
[move_group-4] [WARN] [1719469828.031295433] [move_group.moveit.moveit.ros.occupancy_map_monitor]: Resolution not specified for Octomap. Assuming resolution = 0.1 instead
[move_group-4] [ERROR] [1719469828.031306051] [move_group.moveit.moveit.ros.occupancy_map_monitor]: No 3D sensor plugin(s) defined for octomap updates
[move_group-4] terminate called after throwing an instance of 'rclcpp::ParameterTypeException'
[move_group-4]   what():  expected [string_array] got [string]
[move_group-4] Stack trace (most recent call last):
[move_group-4] #17   Object "", at 0xffffffffffffffff, in 
[INFO] [spawner-8]: process has finished cleanly [pid 5767]
[move_group-4] #16   Object "/home/daniel/ws_moveit/install/moveit_ros_move_group/lib/moveit_ros_move_group/move_group", at 0x61344fb181a4, in _start
[move_group-4] #15   Source "../csu/libc-start.c", line 360, in __libc_start_main_impl [0x70630162a28a]
[move_group-4] #14   Source "../sysdeps/nptl/libc_start_call_main.h", line 58, in __libc_start_call_main [0x70630162a1c9]
[move_group-4] #13   Object "/home/daniel/ws_moveit/install/moveit_ros_move_group/lib/moveit_ros_move_group/move_group", at 0x61344fb17199, in main
[move_group-4] #12   Object "/home/daniel/ws_moveit/install/moveit_ros_planning/lib/libmoveit_cpp.so.2.10.0", at 0x7063024b25e1, in moveit_cpp::MoveItCpp::MoveItCpp(std::shared_ptr<rclcpp::Node> const&, moveit_cpp::MoveItCpp::Options const&)
[move_group-4] #11   Object "/home/daniel/ws_moveit/install/moveit_ros_planning/lib/libmoveit_cpp.so.2.10.0", at 0x7063024b06e7, in moveit_cpp::MoveItCpp::loadPlanningPipelines(moveit_cpp::MoveItCpp::PlanningPipelineOptions const&)
[rviz2-1] [INFO] [1719469828.345185857] [rviz2]: Stereo is NOT SUPPORTED
[rviz2-1] [INFO] [1719469828.345268351] [rviz2]: OpenGl version: 4.6 (GLSL 4.6)
[move_group-4] #10   Object "/home/daniel/ws_moveit/install/moveit_ros_planning/lib/libmoveit_planning_pipeline_interfaces.so.2.10.0", at 0x7063019ba0c7, in moveit::planning_pipeline_interfaces::createPlanningPipelineMap(std::vector<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::allocator<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > > > const&, std::shared_ptr<moveit::core::RobotModel const> const&, std::shared_ptr<rclcpp::Node> const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)
[move_group-4] #9    Object "/home/daniel/ws_moveit/install/moveit_ros_planning/lib/libmoveit_planning_pipeline.so.2.10.0", at 0x70630159b1e9, in planning_pipeline::PlanningPipeline::PlanningPipeline(std::shared_ptr<moveit::core::RobotModel const> const&, std::shared_ptr<rclcpp::Node> const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)
[move_group-4] #8    Object "/home/daniel/ws_moveit/install/moveit_ros_planning/lib/libmoveit_planning_pipeline.so.2.10.0", at 0x7063015da584, in planning_pipeline_parameters::ParamListener::declare_params()
[move_group-4] #7    Object "/opt/ros/jazzy/lib/librclcpp.so", at 0x706301ed4817, in 
[rviz2-1] [INFO] [1719469828.447808079] [rviz2]: Stereo is NOT SUPPORTED
[move_group-4] #6    Source "../../../../src/libstdc++-v3/libsupc++/eh_throw.cc", line 98, in __cxa_throw [0x706301abb127]
[move_group-4] #5    Source "../../../../src/libstdc++-v3/libsupc++/eh_terminate.cc", line 58, in terminate [0x706301aa5a48]
[move_group-4] #4    Source "../../../../src/libstdc++-v3/libsupc++/eh_terminate.cc", line 48, in __terminate [0x706301abae9b]
[move_group-4] #3    Source "../../../../src/libstdc++-v3/libsupc++/vterminate.cc", line 95, in __verbose_terminate_handler [0x706301aa5ffd]
[move_group-4] #2    Source "./stdlib/abort.c", line 79, in abort [0x7063016288fe]
[move_group-4] #1    Source "../sysdeps/posix/raise.c", line 26, in raise [0x70630164526d]
[move_group-4] #0  | Source "./nptl/pthread_kill.c", line 89, in __pthread_kill_internal
[move_group-4]     | Source "./nptl/pthread_kill.c", line 78, in __pthread_kill_implementation
[move_group-4]       Source "./nptl/pthread_kill.c", line 44, in __pthread_kill [0x70630169eb1c]
[move_group-4] Aborted (Signal sent by tkill() 5763 1000)
[rviz2-1] [WARN] [1719469828.527012901] [rcl.logging_rosout]: Publisher already registered for node name: 'rviz2'. If this is due to multiple nodes with the same name then all logs for the logger named 'rviz2' will go out over the existing publisher. As soon as any node with that name is destructed it will unregister the publisher, preventing any further logs for that name from being published on the rosout topic.
[rviz2-1] Warning: class_loader.impl: SEVERE WARNING!!! A namespace collision has occurred with plugin factory for class rviz_default_plugins::displays::InteractiveMarkerDisplay. New factory will OVERWRITE existing one. This situation occurs when libraries containing plugins are directly linked against an executable (the one running right now generating this message). Please separate plugins out into their own library or just don't link against the library and use either class_loader::ClassLoader/MultiLibraryClassLoader to open.
[rviz2-1]          at line 321 in /opt/ros/jazzy/include/class_loader/class_loader/class_loader_core.hpp
[ros2_control_node-5] [INFO] [1719469829.008338426] [controller_manager]: Loading controller 'joint_state_broadcaster'
[spawner-6] [INFO] [1719469829.041360679] [spawner_joint_state_broadcaster]: Loaded joint_state_broadcaster
[ros2_control_node-5] [INFO] [1719469829.041958329] [controller_manager]: Configuring controller 'joint_state_broadcaster'
[ros2_control_node-5] [INFO] [1719469829.042001410] [joint_state_broadcaster]: 'joints' or 'interfaces' parameter is empty. All available state interfaces will be published
[spawner-6] [INFO] [1719469829.046166480] [spawner_joint_state_broadcaster]: Configured and activated joint_state_broadcaster
[ros2_control_node-5] [INFO] [1719469829.090013560] [controller_manager]: Loading controller 'joint_trajectory_controller'
[spawner-7] [INFO] [1719469829.111669583] [spawner_joint_trajectory_controller]: Loaded joint_trajectory_controller
[ros2_control_node-5] [INFO] [1719469829.112063870] [controller_manager]: Configuring controller 'joint_trajectory_controller'
[ros2_control_node-5] [INFO] [1719469829.112122056] [joint_trajectory_controller]: No specific joint names are used for command interfaces. Using 'joints' parameter.
[ros2_control_node-5] [INFO] [1719469829.112132227] [joint_trajectory_controller]: Command interfaces are [position] and state interfaces are [position velocity].
[ros2_control_node-5] [INFO] [1719469829.112135429] [joint_trajectory_controller]: Using 'splines' interpolation method.
[ros2_control_node-5] [INFO] [1719469829.112887363] [joint_trajectory_controller]: Action status changes will be monitored at 20.00 Hz.
[spawner-7] [INFO] [1719469829.118387614] [spawner_joint_trajectory_controller]: Configured and activated joint_trajectory_controller
[INFO] [spawner-6]: process has finished cleanly [pid 5765]
[INFO] [spawner-7]: process has finished cleanly [pid 5766]
[ERROR] [move_group-4]: process has died [pid 5763, exit code -6, cmd '/home/daniel/ws_moveit/install/moveit_ros_move_group/lib/moveit_ros_move_group/move_group --ros-args --params-file /tmp/launch_params_8wv1ozvj'].
[rviz2-1] [ERROR] [1719469831.561333375] [rviz2.moveit.ros.motion_planning_frame]: Action server: /recognize_objects not available
[rviz2-1] [INFO] [1719469831.583174378] [rviz2.moveit.ros.motion_planning_frame]: MoveGroup namespace changed: / -> . Reloading params.
[rviz2-1] [INFO] [1719469831.591040569] [rviz2.moveit.ros.rdf_loader]: Loaded robot model in 0.00243219 seconds
[rviz2-1] [INFO] [1719469831.591084732] [rviz2.moveit.core.robot_model]: Loading robot model 'gen3'...
[rviz2-1] [INFO] [1719469831.591094913] [rviz2.moveit.core.robot_model]: No root/virtual joint specified in SRDF. Assuming fixed joint
[rviz2-1] [INFO] [1719469831.792460710] [rviz2.moveit.ros.rdf_loader]: Loaded robot model in 0.00246267 seconds
[rviz2-1] [INFO] [1719469831.792496691] [rviz2.moveit.core.robot_model]: Loading robot model 'gen3'...
[rviz2-1] [INFO] [1719469831.792507472] [rviz2.moveit.core.robot_model]: No root/virtual joint specified in SRDF. Assuming fixed joint
[rviz2-1] [INFO] [1719469831.839831882] [rviz2.moveit.kinematics.kdl_kinematics_plugin]: Joint weights for group 'manipulator': 1 1 1 1 1 1 1
[rviz2-1] [INFO] [1719469831.999048264] [rviz2.moveit.ros.planning_scene_monitor]: Starting planning scene monitor
[rviz2-1] [INFO] [1719469831.999530693] [rviz2.moveit.ros.planning_scene_monitor]: Listening to '/monitored_planning_scene'
[rviz2-1] [INFO] [1719469832.000917322] [rviz2.moveit.ros.planning_scene_monitor]: Starting planning scene monitor
[rviz2-1] [INFO] [1719469832.001644092] [rviz2.moveit.ros.planning_scene_monitor]: Listening to '/monitored_planning_scene'
[rviz2-1] [INFO] [1719469832.008552172] [interactive_marker_display_95384090651984]: Connected on namespace: /rviz_moveit_motion_planning_display/robot_interaction_interactive_marker_topic
[rviz2-1] [INFO] [1719469832.010610181] [rviz2.moveit.ros.robot_interaction]: No active joints or end effectors found for group 'gripper'. Make sure that kinematics.yaml is loaded in this node's namespace.
[rviz2-1] [INFO] [1719469832.011030212] [rviz2.moveit.ros.robot_interaction]: No active joints or end effectors found for group 'gripper'. Make sure that kinematics.yaml is loaded in this node's namespace.
[rviz2-1] [INFO] [1719469832.031785004] [interactive_marker_display_95384090651984]: Sending request for interactive markers
[rviz2-1] [INFO] [1719469832.063473043] [interactive_marker_display_95384090651984]: Service response received for initialization
[rviz2-1] [INFO] [1719469837.001531115] [rviz2.moveit.ros.planning_scene_monitor]: Failed to call service get_planning_scene, have you launched move_group or called psm.providePlanningSceneService()?
[rviz2-1] [INFO] [1719469837.012758621] [rviz2.moveit.ros.planning_scene_monitor]: Failed to call service get_planning_scene, have you launched move_group or called psm.providePlanningSceneService()?
[rviz2-1] [INFO] [1719469837.012907910] [rviz2.moveit.ros.motion_planning_frame]: group gripper
[rviz2-1] [INFO] [1719469837.012922506] [rviz2.moveit.ros.motion_planning_frame]: Constructing new MoveGroup connection for group 'gripper' in namespace ''
```

**Comment by manadaniel** [comment]
> I confirm that I am experiencing the same. It's the same issue as [here](https://github.com/moveit/moveit2/issues/2854#issuecomment-2152224079) and this solution solved it for me.

Awesome! The proposed solution solved the problem in my case, too. Thx!

**Comment by JenniferBuehler** [comment]
Glad to hear!
Remaining issue on the demo for me: I cannot see the interactive markers. I'm looking for a solution now, and will open a separate issue if there isn't any, but since you're working on this right now too I thought it's worth checking if  the markers are also not working for you?

**Comment by JenniferBuehler** [comment]
Never mind I overlooked that I have to change the planning group to "manipulator". All working now :)

**Comment by Leopold-Klotz** [comment]
I was not able to find success in proposed solutions above. This issue seems to better match the errors that I was working through. [Moveit2_tutorials issue 910](https://github.com/moveit/moveit2_tutorials/issues/910)

While I didn't find a solution to the specific issue, the following process ended up leading to a different result. I reinstalled from binary, as well as CycloneDDS middle ware. When following the MoveIt2 tutorials, I also had to specify `git clone --branch humble` ... when cloning the tutorial code. 
This ended up clearing the problem that I was having, hopefully it works similarly for others.
